src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
+                                    + "| |_| | |_| |   <  __/\n"
+                                    + "|____/ \\__,_|_|\\_\\___|\n";
+
+    static final String dukeGreeting =
+            String.format("%s%s\t Hello! I'm Duke - your personal task manager \n\t%s%s",
+                    LOGO,
+                    lineDivider,
+                    dukeKeywords,
+                    lineDivider);
+    static final String dukeFarewell = String.format("%s\t Bye. Hope to see you again soon!\n%s",
+            lineDivider,
+            lineDivider);
+
+    static Scanner sc = new Scanner(System.in);
+
+    public static void echos(){
+        String userInput = sc.nextLine();
+        while (!userInput.toLowerCase().equals("bye")){
+            String echos = String.format("%s\t %s\n%s",lineDivider,userInput.toLowerCase(),lineDivider);
+            System.out.println(echos);
+            userInput = sc.nextLine();
+        }
+    }
+
+
+    public static void taskApp(){
+        ArrayList<Task> taskList = new ArrayList<>();
+        while (true){
+            System.out.print("Enter command: ");
+            String userInput = sc.nextLine();
+
+            String commandWord = userInput.split(" ")[0].toLowerCase();
+            String content = userInput.substring(userInput.indexOf(" ") + 1);
+
+            switch (commandWord) {
+                case "todo":
+                    System.out.println("todo");
+                    taskList.add(new Todo(content));
+                    addTaskSuccessMessage(taskList, "\tGot it. I've added this task: ");
+                    break;
+                    
+                case "deadline":
+                    System.out.println("deadline");
+                    String deadlineTask = content.split("/")[0].toLowerCase();
+                    String datelineDateBy = content.split("/")[1].split(" ")[1];
+                    taskList.add(new Deadline(deadlineTask,datelineDateBy ));
+                    addTaskSuccessMessage(taskList, "\tGot it. I've added this deadline: ");
+                    break;
+                    
+                case "event":
+                    System.out.println("event");
+                    String eventTask = content.split("/")[0].toLowerCase();
+                    String eventDateBy = content.split("/")[1];
+                    eventDateBy = eventDateBy.substring(eventDateBy.indexOf(" "));
+                    taskList.add(new Event(eventTask,eventDateBy));
+                    addTaskSuccessMessage(taskList, "\tGot it. I've added this Event: ");
+                    break;
+
+                case "list":
+
+                    String listReturnString = String.format("%s%s%s",lineDivider,getList(taskList),lineDivider);
+                    System.out.println(listReturnString);
+
+                    break;
+
+                case "done":
+                    markTaskDone(taskList, userInput, content);
+
+                    break;
```

Comment:
Maybe the number of blank lines at the start and end of each case can be consistent?